### ISSUE TITLE
qgis2threejs not compatible with QGIS 3.0.1-Girona?

### ISSUE THREAD

**Issue by gcutt** [minorua/Qgis2threejs#125]
Qgis2threejs plugin not able to be installed in QGIS 3.0.1.  This plugin is not available for installation from the repository, and now must be installed using downloaded zip file, but that also leads to errors and fails, e.g.:

Couldn't load plugin 'Qgis2threejs' due to an error when calling its classFactory() method 

ModuleNotFoundError: No module named 'qgis2threejs' 
Traceback (most recent call last):
  File "C:/PROGRA~1/QGIS3~1.0/apps/qgis/./python\qgis\utils.py", line 336, in startPlugin
    plugins[packageName] = package.classFactory(iface)
  File "C:/Users/mm/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\Qgis2threejs\__init__.py", line 27, in classFactory
    from qgis2threejs import Qgis2threejs
  File "C:/PROGRA~1/QGIS3~1.0/apps/qgis/./python\qgis\utils.py", line 664, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
ModuleNotFoundError: No module named 'qgis2threejs'

Python version: 3.6.0 (v3.6.0:41df79263a11, Dec 23 2016, 08:06:12) [MSC v.1900 64 bit (AMD64)] 
QGIS version: 3.0.1-Girona Girona, a86bec25eb 

Python Path:
C:/PROGRA~1/QGIS3~1.0/apps/qgis/./python
C:/Users/mm/AppData/Roaming/QGIS/QGIS3\profiles\default/python
C:/Users/mm/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins
C:/PROGRA~1/QGIS3~1.0/apps/qgis/./python/plugins
C:\Program Files\QGIS 3.0\bin\python36.zip
C:\PROGRA~1\QGIS3~1.0\apps\Python36\DLLs
C:\PROGRA~1\QGIS3~1.0\apps\Python36\lib
C:\Program Files\QGIS 3.0\bin
C:\PROGRA~1\QGIS3~1.0\apps\Python36
C:\PROGRA~1\QGIS3~1.0\apps\Python36\lib\site-packages
C:/Users/mm/AppData/Roaming/QGIS/QGIS3\profiles\default/python

**Comment by DigDigDig** [comment]
Minorua is in the process of updating the plugin to version 2.0.  

Until it's released then I recommend using a 2.xx version of QGIS with the 1.xx version of Qgis2threejs

**Comment by lhilarides** [comment]
Just FYI, I run into the same issue when manually installing the plugin from a zipfile on a Windows 10 machine running QGis 3.0.1 
 ModuleNotFoundError: No module named 'qgis2threejs' )

**Comment by minorua** [comment]
The top directory name included in the zip archive downloaded from GitHub site is `Qgis2threejs-master`, so you need to rename it to `Qgis2threejs`.
```
from: {User directory}\AppData\Roaming\QGIS\QGIS3\profiles\default\python\plugins\Qgis2threejs-master
  to: {User directory}\AppData\Roaming\QGIS\QGIS3\profiles\default\python\plugins\Qgis2threejs
```

**Comment by GuillemHerrera** [comment]
Now this amazing plugin is still avalaible from oficial repos for QGIS 3.0.1, but seems like have some code error. 

I can't be able to run it. It's correctly instaled, but show me a error when I launch it: 
```
TypeError: arguments did not match any overloaded call:
  QAction(QObject): argument 1 has unexpected type 'str'
  QAction(str, QObject): not enough arguments
  QAction(QIcon, str, QObject): argument 1 has unexpected type 'str'

```
Thanks for you work! I hope that I can enjoy the plugin early!

**Comment by minorua** [comment]
@GuillemHerrera give me a full stack trace text.

**Comment by JCCabreraMX** [comment]
@GuillemHerrera The same happens to me and I don't know why. 

The complete error text that I get is this:

`An error occurred while executing the Python code:

TypeError: arguments did not match any overloaded call:   
QAction(QObject): argument 1 has unexpected type 'str'   
QAction(str, QObject): not enough arguments   
QAction(QIcon, str, QObject): argument 1 has unexpected type 'str' 

Traceback (most recent call last):
  File "/home/user/.local/share/QGIS/QGIS3/profiles/default/python/plugins/Qgis2threejs/qgis2threejs.py", line 104, in openExporterWithPreviewDisabled
    self.openExporter(False, False)
  File "/home/user/.local/share/QGIS/QGIS3/profiles/default/python/plugins/Qgis2threejs/qgis2threejs.py", line 97, in openExporter
    self.liveExporter = Q3DWindow(self.iface.mainWindow(), self.iface, self.controller, isViewer=True, preview=preview)
  File "/home/user/.local/share/QGIS/QGIS3/profiles/default/python/plugins/Qgis2threejs/q3dwindow.py", line 167, in __init__
    self.ui.setupUi(self)
  File "/home/user/.local/share/QGIS/QGIS3/profiles/default/python/plugins/Qgis2threejs/ui/q3dwindow.py", line 56, in setupUi
    self.treeView = Q3DTreeView(self.dockWidgetContents)
  File "/home/user/.local/share/QGIS/QGIS3/profiles/default/python/plugins/Qgis2threejs/q3dtreeview.py", line 45, in __init__
    self.actionProperties = QAction("Properties")
TypeError: arguments did not match any overloaded call:
  QAction(QObject): argument 1 has unexpected type 'str'
  QAction(str, QObject): not enough arguments
  QAction(QIcon, str, QObject): argument 1 has unexpected type 'str'`

**Comment by GuillemHerrera** [comment]
@minorua  Ok, here is it

```
Traceback (most recent call last):
			  File "/home/user/.local/share/QGIS/QGIS3/profiles/default/python/plugins/Qgis2threejs/qgis2threejs.py", line 97, in openExporter
			    self.liveExporter = Q3DWindow(self.iface.mainWindow(), self.iface, self.controller, isViewer=True, preview=preview)
			  File "/home/user/.local/share/QGIS/QGIS3/profiles/default/python/plugins/Qgis2threejs/q3dwindow.py", line 167, in __init__
			    self.ui.setupUi(self)
			  File "/home/user/.local/share/QGIS/QGIS3/profiles/default/python/plugins/Qgis2threejs/ui/q3dwindow.py", line 56, in setupUi
			    self.treeView = Q3DTreeView(self.dockWidgetContents)
			  File "/home/user/.local/share/QGIS/QGIS3/profiles/default/python/plugins/Qgis2threejs/q3dtreeview.py", line 45, in __init__
			    self.actionProperties = QAction("Properties")
			TypeError: arguments did not match any overloaded call:
			  QAction(QObject): argument 1 has unexpected type 'str'
			  QAction(str, QObject): not enough arguments
			  QAction(QIcon, str, QObject): argument 1 has unexpected type 'str'


Python version: 3.5.2 (default, Nov 23 2017, 16:37:01) [GCC 5.4.0 20160609] 
QGIS version: 3.0.1-Girona Girona, a86bec2 

Python Path:
/usr/share/qgis/python
/home/user/.local/share/QGIS/QGIS3/profiles/default/python
/home/user/.local/share/QGIS/QGIS3/profiles/default/python/plugins
/usr/share/qgis/python/plugins
/usr/lib/python35.zip
/usr/lib/python3.5
/usr/lib/python3.5/plat-x86_64-linux-gnu
/usr/lib/python3.5/lib-dynload
/home/user/.local/lib/python3.5/site-packages
/usr/local/lib/python3.5/dist-packages
/usr/lib/python3/dist-packages
/home/user/.local/share/QGIS/QGIS3/profiles/default/python
```
Thanks!

**Comment by aourednik** [comment]
Ubuntu 16.
Python version: 3.5.2 (default, Nov 23 2017, 16:37:01) [GCC 5.4.0 20160609] 
QGIS version: 3.0.1-Girona Girona, a86bec2 

@GuillemHerrera @minorua Exactly the same error traceback for me, ending in: 

```
TypeError: arguments did not match any overloaded call:
  QAction(QObject): argument 1 has unexpected type 'str'
  QAction(str, QObject): not enough arguments
  QAction(QIcon, str, QObject): argument 1 has unexpected type 'str'
```

**Comment by GuillemHerrera** [comment]
Now is available the plugin version 2.0.1 and it resolve my issues. 
Works very fine and I'm now trying the new capabilities.

Thanks again for this great plugin and work

**Comment by minorua** [comment]
@hectorn61 Fixed in 2ad3f62. Thanks.
